### ISSUE TITLE
EXFAT support also for HDD?

### ISSUE THREAD

**Issue by well0nez** [israpps/wLaunchELF_ISR#48]
Hello, thank you for your nice tool and ongoing work on it.
I use an HDD with exfat for playing with opl, does wLaunchELF also support extfat for HDDs ?

**Comment by israpps** [comment]
> Hello, thank you for your nice tool and ongoing work on it. I use an HDD with exfat for playing with opl, does wLaunchELF also support extfat for HDDs ?
> 
> I use grimdoomers OPL to play via HDD with extfat: https://github.com/grimdoomer/Open-PS2-Loader

not on my plans for now...

**Comment by well0nez** [comment]
Oh, very sad to hear.
Thank you anyway! 
Keep up the good work

**Comment by israpps** [comment]
> Oh, very sad to hear. Thank you anyway! Keep up the good work

ill probably make it, but in a few months. dont have time now. because what you ask implies disabling all the already existing HDD code...

**Comment by well0nez** [comment]
Thank you, i want to have a way to copy games from a exfat USB-Device to my exfat HDD without the need of a computer or a reason to uninstall the HDD from the PS2. i havent heard of a way to do this.

**Comment by israpps** [comment]
> Thank you, i want to have a way to copy games from a exfat USB-Device to my exfat HDD without the need of a computer or a reason to uninstall the HDD from the PS2. i havent heard of a way to do this.

That will take 8 hours per game or so...

**Comment by well0nez** [comment]
> > Thank you, i want to have a way to copy games from a exfat USB-Device to my exfat HDD without the need of a computer or a reason to uninstall the HDD from the PS2. i havent heard of a way to do this.
> 
> That will take 8 hours per game or so...

Thats okay, i have no access to a computer there, so better than nothing.

**Comment by well0nez** [comment]
I just send you some bucks, thanks for your ongoing work

**Comment by israpps** [comment]
> I just send you some bucks, thanks for your ongoing work PS: I couldnt send it via friends, it said that you cannot recieve personal payments

Thanks!

**Comment by Arnold101knight** [comment]
> > Hello, thank you for your nice tool and ongoing work on it. I use an HDD with exfat for playing with opl, does wLaunchELF also support extfat for HDDs ?
> > I use grimdoomers OPL to play via HDD with extfat: https://github.com/grimdoomer/Open-PS2-Loader
> 
> not on my plans for now...

WHY???????????????????!!!!!!!!!!!!!!!!!!! WE NEED IT!!!!!!

**Comment by israpps** [comment]
> > > Hello, thank you for your nice tool and ongoing work on it. I use an HDD with exfat for playing with opl, does wLaunchELF also support extfat for HDDs ?
> > > I use grimdoomers OPL to play via HDD with extfat: https://github.com/grimdoomer/Open-PS2-Loader
> > 
> > 
> > not on my plans for now...
> 
> WHY???????????????????!!!!!!!!!!!!!!!!!!! WE NEED IT!!!!!!

Because I'm busy with other things

Right now, I'm trying to make wle fully compatible with namco arcade systems. Maybe I'll do this next.... Not sure...

**Comment by Arnold101knight** [comment]
you said 1 year ago that you had done in few months....

also, ps2 is used by a lot of people, arcade system by who? 10 people in world who own it? ps2 comes first

from when we formatted it with exfat we can't load apps and ps1 games from the hard drive. Please do it :(

not sure why???

**Comment by AcidReach** [comment]
This feature would be very very helpful.

**Comment by israpps** [comment]
> also, ps2 is used by a lot of people, arcade system by who? 10 people in world who own it? ps2 comes first

There's a lot of people who own arcade PS2s... And whatever I work on, it's up to me...

Also FYI, I tried to make the EXFAT HDD support some months ago and the driver for EXFAT HDD just crashes, not even the exception handler gives information to work with at debugging it... So I dropped it

> from when we formatted it with exfat we can't load apps and ps1 games from the hard drive. Please do it :(

Popstarter is not compatible with EXFAT HDD and never will, the program is closed source

**Comment by israpps** [comment]
> This feature would be very very helpful.

Yes. But I couldn't get it working. And since the crash is happening on the PS1 CPU, I can't use a debugger to help unless I get my hands on a devkit (impossible to get where I live)

**Comment by AcidReach** [comment]
> > This feature would be very very helpful.
> 
> Yes. But I couldn't get it working. And since the crash is happening on the PS1 CPU, I can't use a debugger to help unless I get my hands on a devkit (impossible to get where I live)

Thanks for the message and sorry you couldn’t get it working yet. 👍🏻

**Comment by israpps** [comment]
imposible to debug this

Exception handlers throw nothing on real hardware, PCSX2 crashes altogether

This feature will take a loooooong time, maybe it will never happen

**Comment by Immersion95** [comment]
Found this fork based on yours that works : https://github.com/saildot4k/wLaunchELF_R3Z

**Comment by israpps** [comment]
> Found this fork based on yours that works : https://github.com/saildot4k/wLaunchELF_R3Z

Yes, r3z3n moved my wLaunchELF to latest SDK 

That seemed to be the only way to make it happen
